.github/workflows/release.yml

```diff
@@ -206,6 +206,13 @@ jobs:
           REPO_NAME: twilio/twilio-cli
           REPO_ACCESS_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
           BRANCH_NAME: ${{steps.extract_branch.outputs.branch}}
+      - name: Run acceptance testing for docker release
+        run: node .github/scripts/trigger-workflow.js
+        env:
+          WORKFLOW_NAME: '.github/workflows/cli-test-docker.yml'
+          REPO_NAME: twilio/twilio-cli
```

Comment:
Nit: Can we change the hard-coding?

Reply:
Sure, done. For other occurrences of the str in release workflow, can be picked up in [DII-309](https://issues.corp.twilio.com/browse/DII-309).